### PR TITLE
Add initial amp-springboard-player sample

### PR DESCRIPTION
**Pull request by jmadler**
#252

### CODE REVIEW

File: src/20_Components/amp-springboard-player.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  #### Introduction
+
+  Embed videos hosted on the Springboard Video Platform in your AMP HTML files.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/components/amp-vimeo/">
```
Comment (kul3r4) [inline src/20_Components/amp-springboard-player.html]: amp-vimeo -> amp-springboard-player

File: src/20_Components/amp-springboard-player.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+    Embed videos hosted on the Springboard Video Platform via the `data-site-id`, `data-domain`, `data-player-id`, `data-content-id` attributes.
+    The `data-mode` attribute is used to determine if this is a single video or playlist.  If it's a playlist, the `data-items` attribute specifies the number of items in the playlist.
```
Comment (kul3r4) [inline src/20_Components/amp-springboard-player.html]: Can we add an example for a playlist if possible?

Comment (jmadler) [inline src/20_Components/amp-springboard-player.html]: I don't have any example content for a playlist unfortunately.  I'll reach out to springboard for a sample playlist.

Comment (kul3r4) [inline src/20_Components/amp-springboard-player.html]: Ok, we can add it later and merge this PR once the tags are fixed.

Comment (jmadler) [inline src/20_Components/amp-springboard-player.html]: SGTM

**Comment by jmadler** [comment]
PTAL

File: src/20_Components/amp-springboard-player.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  #### Introduction
+
+  Embed videos hosted on the Springboard Video Platform in your AMP HTML files.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/components/amp-springboard/">
```
Comment (kul3r4) [inline src/20_Components/amp-springboard-player.html]: is it the name of the page amp-springboard-player? I think "player" is missing

**Comment by kul3r4** [comment]
@jmadler friendly ping

**Comment by sebastianbenz** [comment]
@jmadler ping

File: src/20_Components/amp-springboard-player.html (hunk 3 above)
Comment (jmadler) [inline src/20_Components/amp-springboard-player.html]: thanks!  fixed

**Pull request merged**